src/include/migraphx/compile_src.hpp

```diff
@@ -37,21 +37,25 @@ inline namespace MIGRAPHX_INLINE_NS {
 struct src_file
 {
     fs::path path;
-    std::pair<const char*, const char*> content;
-    std::size_t len() const { return content.second - content.first; }
+    std::string_view content;
+
+    src_file() = delete;
+    src_file(fs::path path_, std::string_view content_)
```

Comment:
These changes should go into a seperate PR.

Reply:
We need those changes to read from embedded resource file.